src/test/java/core/basesyntax/service/strategy/SupplyHandlerTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class SupplyHandlerTest {
+    private Storage storage;
+    private BalanceHandler handler;
+    private SupplyHandler supplyHandler;
+    private final String fruit = "apple";
+    private final int quantity = 20;
+    private final int addQuantity = 200;
+
+    @BeforeEach
+    void setUp() {
+        storage = new Storage();
+        handler = new BalanceHandler(storage);
+        supplyHandler = new SupplyHandler(storage);
+    }
+
+    @Test
+    void put_NewInformationToStorage_Ok() {
```

Comment:
And please take a look on mathouds naming, because if you are waiting on exception in method, you need to name notOk. I didn't look on all methouds naming. That's on you.